lib/main_common.pm

```diff
@@ -567,6 +562,7 @@ sub load_system_role_tests {
     }
 }
 sub load_jeos_tests {
+    # loadtest 'jeos/sccreg';
```

Comment:
leftover ?

Reply:
No, fake schedule to pass travis check.  The module does not exist at all.